src/mono/mono.proj

```diff
@@ -81,4 +81,23 @@
 
   <Target Name="Test" />
 
+  <Target Name="RunSample">
+    <PropertyGroup>
+      <TestHostPath>$(ArtifactsDir)bin/testhost/$(NetCoreAppCurrent)-$(OSGroup)-$(Configuration)-$(Platform)</TestHostPath>
+      <DotNetExec Condition="'$(TargetsWindows)' == 'true'">dotnet.exe</DotNetExec>
+      <DotNetExec Condition="'$(DotNetExec)' == ''">dotnet</DotNetExec>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_MonoRuntimeArtifacts Include="$(BinDir)/*.*" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(_MonoRuntimeArtifacts)"
+          DestinationFolder="$(TestHostPath)/shared/Microsoft.NETCore.App/$(NETCoreAppCurrentVersion).0" 
```

Comment:
```suggestion
          DestinationFolder="$(TestHostPath)/shared/Microsoft.NETCore.App/$(ProductVersion)" 
```

Reply:
oh and all of the forward slashes should be backslashes (coding convention of the msbuild files).